### DISCUSSION TITLE
search doesnot disappear once entring a folder

### DISCUSSION THREAD

**Discussion by anwaruddin2k** [spacedriveapp/spacedrive#1862] [Feature Request]
when I search for folder , once I find what I am looking for search query stays . It gets removed when I do search in nemo cinnamon.

**Comment by utkubakir** [comment]
Can you take a video?

**Comment by anwaruddin2k** [comment]
https://github.com/spacedriveapp/spacedrive/assets/53415502/8c4ee053-cf30-49d6-82f4-1448493c4980


https://github.com/spacedriveapp/spacedrive/assets/53415502/894f04ad-65d2-415d-83b5-17c9136adeff

**Comment by utkubakir** [comment]
https://github.com/spacedriveapp/spacedrive/pull/1890

Thank you for reporting!
